bpo-34160: Make sorting attributes in XML serialization optional. (#10452)
python/cpython: Doc/library/xml.dom.minidom.rst

   For the :class:`Document` node, an additional keyword argument *encoding* can
   be used to specify the encoding field of the XML header.

>    If *sort_attrs* is true (by default), then element attributes will be

Reviewer: ```suggestion
   If *sort_attrs* is ``True`` (the default value), then element attributes will be
```

Author: No, the current wording is more correct. Element attributes will be sorted not only when *sort_attrs* is True, but when its boolean value is true.

"True" is a concrete bool instance, "true" is used when the boolean value should be true, and not just bool instances are accepted.

   encoding. Encoding this string in an encoding other than UTF-8 is
   likely incorrect, since UTF-8 is the default encoding of XML.

>    The *sort_attrs* argument behaves like the corresponding arguments of

Author: ```suggestion
   The *sort_attrs* argument behaves like the corresponding argument of
```

>    :meth:`writexml`.

Reviewer 2: Singular: _corresponding argument_

   .. versionchanged:: 3.8
      Added support for the *sort_attrs* argument.

.. method:: Node.toprettyxml(indent="", newl="", encoding="", *, sort_attrs=True)

   Return a pretty-printed version of the document. *indent* specifies the
   indentation string and defaults to a tabulator; *newl* specifies the string
   emitted at the end of each line and defaults to ``\n``.

>    The *encoding* and *sort_attrs* argument behave like the corresponding

Reviewer: ```suggestion
   The *encoding* and *sort_attrs* arguments behave like the corresponding
```
